portal/src/components/Typography/Typography.tsx

```diff
@@ -50,6 +50,24 @@ export const Paragraph: React.FC = ({ children, ...rest }) => (
     </p>
 );
 
+export const Blockquote: React.FC = ({ children, ...rest }) => {
+    const child: ReactElement = React.Children.only(children) as ReactElement;
+
+    if (!child.props) {
```

Comment:
Usikker på om vi trenger være så defensive her. Er det tatt utgangspunkt i `CodeBlock`? Der er det vel den ekstra koden for å hente ut språk-attributtet, noe vi ikke har for en blockquote.

```tsx
export const Blockquote: React.FC = ({ children, ...rest }) => (
    <blockquote className="jkl-portal-blockquote" {...rest}>
        {children}
    </blockquote>
);
```

Reply:
Grunnen til at det er gjort slik her er at man ellers får en paragraph inne i `blockquote` som da overstyrer stilen igjen, siden den blir formattert med `jkl-body` 😑